packages/dev-server/README.md

```diff
@@ -28,6 +28,13 @@ module.exports = createConfig([
 ```
 
 
+## Pitfalls
+
+Make sure you use this `dev-server` block after your `entryPoint()` blocks. The dev server will extend all of them with the `'webpack/hot/only-dev-server'` entry point by default.
```

Comment:
Are you still able to order them randomly after we did the entry point normalization? Since we now take into account the existing entry points it should not work this way anymore, I think.